reorder_python_imports.py

```diff
@@ -434,7 +434,6 @@ def _fix_file(
     'typing_extensions=typing:TYPE_CHECKING',
     'typing_extensions=typing:Text',
     'typing_extensions=typing:Type',
-    'typing_extensions=typing:get_type_hints',
```

Comment:
please read the PR I linked you to and the comment directly above this

Reply:
Are you referring to a comment in the code, or a comment in the linked PR? Either way I'm not sure what you mean, but I did make one change I noticed that the linked PR did too.

Reply:
the comment in the code that says "this is generated" -- aka you shouldn't edit it manually

Reply:
I've removed the changes to generated code.